gnomad/utils/constraint.py

```diff
@@ -206,12 +208,13 @@ def count_variants_by_group(
             pop,
             pop,
         )
-        agg[f"downsampling_counts_{pop}"] = downsampling_counts_expr(
+        agg[f"downsampling_counts_{pop}"] = pop_counts_expr(
```

Comment:
If skipping downsamplings, we probably don't want to name it `downsampling_counts`

Reply:
Also, what if both downsamplings and a split by pop is wanted? It seems like this should be an addition rather than a skip